doc/sphinx/source/changelog.rst

```diff
@@ -4,6 +4,100 @@ Changelog
+-  Fix bug when using log-scale y-axis for ocean transects. (`#2862 <https://github.com/ESMValGroup/ESMValTool/pull/2862>`__) `Tomas Torsvik <https://github.com/TomasTorsvik>`__
+
+Community
+~~~~~~~~~
+
+-  Add MO-paths to config file (`#2784 <https://github.com/ESMValGroup/ESMValTool/pull/2784>`__) `mo-tgeddes <https://github.com/mo-tgeddes>`__
+
+Deprecations
+~~~~~~~~~~~~
+
+-  Recipe `recipe_esacci_oc.yml` replace with new regrid scheme `nearest_extrapolate` (`#2841 <https://github.com/ESMValGroup/ESMValTool/pull/2841>`__) `Valeriu Predoi <https://github.com/valeriupredoi>`__
+
+Documentation
+~~~~~~~~~~~~~
+
+-  Update release schedule for v2.7 (`#2747 <https://github.com/ESMValGroup/ESMValTool/pull/2747>`__) `Bouwe Andela <https://github.com/bouweandela>`__
+-  Add Met Office installation method (`#2751 <https://github.com/ESMValGroup/ESMValTool/pull/2751>`__) `mo-tgeddes <https://github.com/mo-tgeddes>`__
+-  Add release dates for 2023 (`#2769 <https://github.com/ESMValGroup/ESMValTool/pull/2769>`__) `Rémi Kazeroni <https://github.com/remi-kazeroni>`__
+-  Made `maintainer` entry mandatory for published recipes (`#2703 <https://github.com/ESMValGroup/ESMValTool/pull/2703>`__) `Manuel Schlund <https://github.com/schlunma>`__
+-  Use command with current command line opts for `cffconvert` in documentation (`#2791 <https://github.com/ESMValGroup/ESMValTool/pull/2791>`__) `Valeriu Predoi <https://github.com/valeriupredoi>`__
+-  Update CMORizer documentation with command options (`#2795 <https://github.com/ESMValGroup/ESMValTool/pull/2795>`__) `Rémi Kazeroni <https://github.com/remi-kazeroni>`__
+-  Fixed broken link for monthly meetings (`#2806 <https://github.com/ESMValGroup/ESMValTool/pull/2806>`__) `Rémi Kazeroni <https://github.com/remi-kazeroni>`__
+-  Update MO obs4MIPs paths in the user configuration file (`#2813 <https://github.com/ESMValGroup/ESMValTool/pull/2813>`__) `mo-tgeddes <https://github.com/mo-tgeddes>`__
+-  Fix Windows incompatible file names in documentation of recipe_climate_change_hotspot.yml (`#2823 <https://github.com/ESMValGroup/ESMValTool/pull/2823>`__) `Lee de Mora <https://github.com/ledm>`__
+-  Update documentation for the Landschuetzer 2016 recipe. (`#2801 <https://github.com/ESMValGroup/ESMValTool/pull/2801>`__) `Tomas Torsvik <https://github.com/TomasTorsvik>`__
+-  Fixed anaconda badge in README (`#2866 <https://github.com/ESMValGroup/ESMValTool/pull/2866>`__) `Valeriu Predoi <https://github.com/valeriupredoi>`__
+-  Update release strategy notes (`#2734 <https://github.com/ESMValGroup/ESMValTool/pull/2734>`__) `sloosvel <https://github.com/sloosvel>`__
+-  Add documention on how to handle CMORizers for multiple dataset versions (`#2730 <https://github.com/ESMValGroup/ESMValTool/pull/2730>`__) `Rémi Kazeroni <https://github.com/remi-kazeroni>`__
+-  Extending documentation: broken recipe policy (`#2720 <https://github.com/ESMValGroup/ESMValTool/pull/2720>`__) `Axel Lauer <https://github.com/axel-lauer>`__
```

Comment:
I think #2720 could be removed from the changelog, otherwise the "broken recipe policy" is mentioned twice. Note that the branch of #2720 was merged into the one of #2719 and the latter one to main.

Reply:
done this too! :beer: